dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/servercontainer/v10/ServerContainerImplFactory10.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.wsoc.servercontainer.v10;
+
+import com.ibm.ws.wsoc.servercontainer.ServerContainerExt;
+import com.ibm.ws.wsoc.servercontainer.ServletContainerFactory;
+
+/*
+ * Used for Websocket 1.0, 1.1 and 2.0
+ */
+public class ServerContainerImplFactory10 implements ServletContainerFactory {
```

Comment:
I see we have the exact same class: dev/io.openliberty.wsoc.1.0.internal/src/io/openliberty/wsoc/servercontainer10/ServerContainerImplFactory10.java are both of these necessary?

Reply:
You're right. 

I thought I deleted the one under io.openliberty.wsoc.1.0.internal, but I hadn't. It's now removed in the last commit.